stages/org.osbuild.chown

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+"""
+
+import subprocess
+import sys
+import os
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["paths"],
+"properties": {
+  "paths": {
+    "type": "array",
+    "description": "Paths to operate on for changing ownership",
+    "items": {
+      "type": "string"
+    }
+  },
+  "user": {
+    "type": "string"
+  },
+  "group": {
+    "type": "string"
+  },
+  "recursive": {
+    "type": "boolean"
+  }
+}
+"""
+
+def main(tree, options):
+    paths = options["paths"]
+    user = options.get("user", "")
+    group = options.get("group", "")
+    recursive = options.get("recursive", False)
+
+    arguments = [f"{user}:{group}"]
```

Comment:
In case `user` is `--help`, you want this after `--`, don't you?

Reply:
Yeah same as above